helm/promxy-app/templates/configmap.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "resource.default.name" . }}
+  name: {{ include "resource.configmap.name" . }}
```

Comment:
Just out of curiosity why rename it?
Cause then to reference this resource you'd have `configmap/promxy-configmap` which is repetitive.

Reply:
I can change this sure :) I renamed that as we have this for networkpolicies, psp and so on

Reply:
I changed that to resource.default.name